### DISCUSSION TITLE
Пробел в списке треков для экспорта

### DISCUSSION THREAD

**Discussion by Mkhv-2** [Chimildic/YaMuTools#91] [Идея]
Дорогие разработчики,

Меня зовут Михаил, и я являюсь активным пользователем вашего расширения для Google Chrome. Ваше приложение значительно облегчает мою работу и делает пользование ЯндексМузыкой еще более приятным. 

В настоящее время у меня есть одно предложение, которое, как я считаю, может улучшить функциональность и удобство использования вашего расширения. При экспорте треков, информация об исполнителе и название трека разделяются пробелами. Это может вызвать некоторую путаницу, поскольку в названии исполнителя и трека также часто используются пробелы.

Я бы предложил внести небольшое изменение в эту функцию, добавив тире между названием исполнителя и названием трека. Это бы значительно упростило различие между исполнителем и названием трека и сделало бы формат более читабельным и понятным. 

Итак, вместо текущего формата, например "The Beatles Let It Be", формат будет выглядеть следующим образом: "The Beatles - Let It Be". Я думаю, что такой небольшой нюанс сделает расширение еще более удобным для всех пользователей.

Спасибо за внимание к этому предложению. Я надеюсь, что вы найдете его полезным. Я очень ценю вашу работу и с нетерпением жду новых улучшений.

С уважением,  
Михаил.

**Comment by EldarAliev** [comment]
Только в таком подходе надо учитывать подобные случаи:
1.  два исполнителя у одного трека (пример, https://music.yandex.ru/album/23837994/track/108208293) 
2.  в исполнителях записан ещё и тот кто делал ремикс (пример, https://music.yandex.ru/album/19490868/track/95550899)

Возможно ещё есть вариации не встречавшиеся мне...
